webpack/assets/javascripts/react_app/stores/HostsStore.js

```diff
@@ -0,0 +1,46 @@
+import AppDispatcher from '../dispatcher';
+import {ACTIONS} from '../constants';
+import AppEventEmitter from './AppEventEmitter';
+
+// Internal object of Hosts
+const _hosts = {};
+
+class HostsEventEmitter extends AppEventEmitter {
+  constructor() {
+    super();
+  }
+
+  getHostsData(id) {
+    _hosts[id] = _hosts[id] || { data: [] };
+
+    return _hosts[id];
+  }
+}
+
+const HostsStore = new HostsEventEmitter();
+
+HostsStore.setMaxListeners(50);
```

Comment:
the other alternative I could think of in this case is to drop using the store all togher.
for example, a simple solution can be achvied by changing the PowerStatusContainer to:

```js
 componentDidMount() {
    $.ajax({
      url: url,
      dataType: 'json',
      success: function(state) {
        this.setState(...)
      }.bind(this)
    });
  }```

a lot less fancy, but might bite us less until we have a better host store solution?